.eslintrc.js

```diff
@@ -34,13 +34,18 @@ var SHAREDB_RULES = {
   // as-needed quote props are easier to write
   'quote-props': ['error', 'as-needed'],
   'require-jsdoc': 'off',
-  'valid-jsdoc': 'off'
+  'valid-jsdoc': 'off',
+
+  // Required after upgrade to ecmaVersion: 6
+  'no-invalid-this': 'off'
 };
 
 module.exports = {
   extends: 'google',
   parserOptions: {
-    ecmaVersion: 3
+    // Support ES6 imports and exports
+    ecmaVersion: 6,
+    sourceType: 'module'
```

Comment:
Since this repo in general uses ES3 syntax at the moment, I'd recommend doing a glob override to use this custom config just for the new example, instead of for the entire repo:
https://eslint.org/docs/latest/use/configure/configuration-files#configuration-based-on-glob-patterns